Arduino.mk

```diff
@@ -1562,6 +1578,21 @@ generate_assembly: $(OBJDIR)/$(TARGET).s
 generated_assembly: generate_assembly
 		@$(ECHO) "\"generated_assembly\" target is deprecated. Use \"generate_assembly\" target instead\n\n"
 
+.PHONY: tags
+tags:
+	rm -f $(shell pwd)/$(TAGS_FILE)
```

Comment:
i'm a bit worried about this accidentally deleting stuff if $(TAGS_FILE) is undefined, is it just a single file?

if so then ```rm -f $(TAGS_FILE)``` i guess is a bit safer.....?

also is this going to be in the same directory as the Makefile and .ino, or in $(OBJDIR)? i.e. will "make clean" remove it?

Reply:
I agree. We need a guard condition to prevent accidental deletion.